app/templates/auth/settings.html

```
@@ -148,6 +148,7 @@ <h2 class="text-lg font-medium text-gray-900 dark:text-white">{{ _('Units & Valu
                                         <option value="L/100km" {% if current_user.consumption_unit == 'L/100km' %}selected{% endif %}>L/100km</option>
                                         <option value="mpg" {% if current_user.consumption_unit == 'mpg' %}selected{% endif %}>MPG (UK)</option>
                                         <option value="mpg_us" {% if current_user.consumption_unit == 'mpg_us' %}selected{% endif %}>MPG (US)</option>
+                                        <option value="km/L" {% if current_user.consumption_unit == 'km/L' %}selected{% endif %}>km/L</option>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)